### DISCUSSION TITLE
Did you get numbers from hadronic Z-boson selection based on mass?

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#27] [Comments_HIG_23_001] [answered]
@jialin-guo1 Sometime back we discussed to summarize the difference between the Z-boson candidate based on pT and mass. Did you have this information now?

We also got a question from Jan:

L192 do you not preferentially select jets that are in the analysis window (ie pass the mass requirements, either signal or sideband), before choosing the pair with the highest dijet pT?

**Comment by jialin-guo1** [comment]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/4df7730f-ab4f-4e65-88dc-16732fa7d242)
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/f1770401-d3f5-4f56-b474-1d3713601930)
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/5761aaa0-32ed-45ba-b7fb-3d0f0d2889e3)

with pt base, we have more events in 91Gev mass bin. see M(jj) plot

**Reply by ram1123** [reply]
Can you check the gen-match, and see where we have more correct tagging efficiency?

**Reply by jialin-guo1** [reply, marked as answer]
@ram1123 
gun-match efficiency:

**mass base**:
matched one jet: 91.85%
matched two jet: 72.23%

**pt base**:
matched one jet: 93.09%
matched two jet: 72.24%

According to above result. I would like to say the distinction between pt and mass base is negligible. if we have to choose one. pt base has little more match efficiency

**Reply by ram1123** [reply]
Once we are done with other checks we should also make a 2D plot of GEN vs RECO distribution

- pT GEN vs RECO
- Eta GEN vs RECO
